classes/PaymentModule.php

```diff
@@ -1083,7 +1083,7 @@ protected function createOrderCartRules(
 
         // prepare cart calculator to correctly get the value of each cart rule
         $calculator = $cart->newCalculator($order->product_list, $cart->getCartRules(), $order->id_carrier);
-        $calculator->processCalculation($computingPrecision);
+        $calculator->processCalculation(_PS_PRICE_COMPUTE_PRECISION_);
```

Comment:
because of this https://github.com/PrestaShop/PrestaShop/pull/16616 when it's https://github.com/PrestaShop/PrestaShop/pull/16531 on develop branch

Reply:
damn yes, that makes sense